src/System.Collections/tests/Generic/Dictionary/Dictionary.Generic.Tests.netcoreapp.cs

```diff
@@ -252,17 +253,18 @@ public void TrimExcess_Generic_LargeInitialCapacity_TrimReducesSize()
             Assert.Equal(7, dictionary.EnsureCapacity(0));
         }
 
-        [Fact]
-        public void TrimExcess_Generic_TrimToLargerThanExistingCapacity_DoesNothing()
+        [Theory]
+        [InlineData(20)]
+        public void TrimExcess_Generic_TrimToLargerThanExistingCapacity_DoesNothing(int suggestedCapacity)
         {
             var dictionary = new Dictionary<TKey, TValue>();
             int capacity = dictionary.EnsureCapacity(0);
-            dictionary.TrimExcess(20);
+            dictionary.TrimExcess(suggestedCapacity);
```

Comment:
I meant, passing the inlinedata as every case that you have here and then your test will be like:

```cs
[Theory]
[InlineData(20)]
[InlineData(10)]
public void tests(int suggestedCapacity)
{
   var dictionary = new Dictionary<TKey, TValue>();
   int capacity = dictionary.EnsureCapacity(0);
   dictionary.TrimExcess(suggestedCapacity)
   Assert.Equal(capacity, dictionary.EnsureCapacity(0));
}
```

that way you only have one block of code that will be executed with different values, which is what you are currently doing in the tests but duplicating code.

Same for the tests above.

Reply:
The two tests here are a bit different. One takes capacity arg as input to the Dictionary constructor and one doesnt

Reply:
Ha, missed that. Then maybe add more inline data?